Makefile

```
@@ -0,0 +1,33 @@
+.PHONY: build test container-build deploy clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)